format_spec/fragment.md

```diff
@@ -276,6 +282,12 @@ The footer is a simple blob \(i.e., _not a generic tile_\) with the following in
 | Tile maxes offset for attribute/dimension 1 | `uint64_t` | The offset to the generic tile storing the tile maxes for attribute/dimension 1. |
 | … | … | … |
 | Tile maxes offset for attribute/dimension N | `uint64_t` | The offset to the generic tile storing the tile maxes for attribute/dimension N |
+| Tile global order min coordinates offset for dimension 1 | `uint64_t` | _New in version 23_ For sparse arrays, he offset to the generic tile storing the tile global order mins for dimension 1
+| … | … | … |
+| Tile global order min coordinates offset for dimension N | `uint64_t` | _New in version 23_ For sparse arrays, the offset to the generic tile storing the tile global order mins for dimension N
+| Tile global order max coordinates offset for dimension 1 | `uint64_t` | _New in version 23_ For sparse arrays, the offset to the generic tile storing the tile global order maxes for dimension 1
+| … | … | … |
+| Tile global order max coordinates offset for dimension N | `uint64_t` | _New in version 23_ For sparse arrays, the offset to the generic tile storing the tile global order maxes for dimension N
```

Comment:
* I would prefer phrasing it like `The offset to the […] for dimension N. This field exists only on sparse arrays.`.
* We have yet to decide whether this is how we want to store the tiles, or use the extensible footer design described in CORE-345. If we do the latter, and don't want to change this PR's implementation, one proposed solution was to do it in a subsequent PR that will be merged before the next release.
  * The fact that these fields are only written in the footer for sparse arrays, is another reason to pursue the extensible footer design.
    * If we stick with this design, a better and simpler idea would be to always the offsets, and write zeroes for dense arrays.